src/Service/SQS.service.js

```diff
@@ -51,6 +50,42 @@ export default class SQSService extends DependencyAwareClass {
     }
   }
 
+  /**
+   * Returns an SQS client instance
+   */
+  // eslint-disable-next-line class-methods-use-this
+  get sqs() {
+    if (!this.$sqs) {
+      this.$sqs = new AWS.SQS({
+        region: process.env.REGION,
+      });
+    }
+
+    return this.$sqs;
+  }
```

Comment:
Move to a cached getter so:

1. We don't init SQS on file import
2. We can mock it in tests